src/components/Common/ResourceCategoryPicker.tsx

```
@@ -190,9 +185,13 @@ export function ResourceCategoryPicker({
   const getDisplayValue = () => {
     if (isLoadingSelected) {
       return (
-        <div className="flex items-center gap-2">
-          <Loader2 className="h-4 w-4 animate-spin" />
-          <span className="text-gray-500">Loading...</span>
+        <div
+          className="flex items-center gap-2"
+          role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)